### PR TITLE
*: add a priority option to transaction, use high priority for meta

### PR DESCRIPTION
**Pull request by tiancaiamao**
Support priority and use high priority to load schema information.

PTAL @shenli @zimulala @hanfei1991

### CODE REVIEW

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -402,9 +402,23 @@ func (c *twoPhaseCommitter) prewriteSingleBatch(bo *Backoffer, batch batchKeys)
 	}
 }
 
+func getTxnPriority(txn *tikvTxn) pb.CommandPri {
+	if pri := txn.us.GetOption(kv.Priority); pri != nil {
+		switch pri.(int) {
+		case kv.PriorityNormal:
```
Comment (shenli) [inline store/tikv/2pc.go]: You don't need this case.

Comment (tiancaiamao) [inline store/tikv/2pc.go]: I think write it down is better, code as comment.

Comment (shenli) [inline store/tikv/2pc.go]: This will be called by every kv request. Without this branch is still clear enough.

**Comment by shenli** [comment]
LGTM
Please resolve the conflicts.

File: store/tikv/region_request.go [hunk 2]
```diff
@@ -98,6 +98,7 @@ func (s *RegionRequestSender) SendReq(bo *Backoffer, req *tikvrpc.Request, regio
 }
 
 func (s *RegionRequestSender) sendReqToRegion(bo *Backoffer, ctx *RPCContext, req *tikvrpc.Request, timeout time.Duration) (resp *tikvrpc.Response, retry bool, err error) {
+	ctx.KVCtx.Priority = req.Priority
```
Comment (zimulala) [inline store/tikv/region_request.go]: Processing in the function of `SetContext` may be better.

**Comment by tiancaiamao** [comment]
I add a simple test that check request priority field is set.
PTAL @zimulala @zhangjinpeng1987 @disksing @shenli

**Comment by tiancaiamao** [comment]
PTAL @zimulala @zhangjinpeng1987

**Comment by zimulala** [comment]
LGTM

**Pull request merged**